Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -2305,6 +2305,17 @@ void MediaPlayerPrivateGStreamer::configureElementPlatformQuirks(GstElement* ele
 #endif
 #endif
 
+#if USE(WESTEROS_SINK)
```

Comment:
Can we remove this ifdef?

Reply:
It would mean one more runtime string comparison for all the platforms, but the impact of that would be negligible.